### PR TITLE
Donation page

### PR DESCRIPTION
**Pull request by patjouk**
Our donation is a bit sad right now, let's change that!

I removed info on how to give with Paypal: people can use the donor box to pay with their card or paypal account.

- [x] Content
- [x] Update the sponsor section of index.html
- [x] Check Spelling
- [x] Clean the code
- [x] Remove all the "Paypal giving funds" links (not working)

### CODE REVIEW

File: templates/core/donate.html [hunk 1]
```diff
@@ -2,52 +2,139 @@
...
+            <div class="row clearfix">
+                <div class="col-md-6">
+                    <h3>Our mission:</h3>
+                    <p>Django Girls is a volunteer-run <a href="{% url 'core:foundation' %}">registered charity</a>.
+                    We use your generous donations to teach thousands of women to build websites
+                    using Python and Django by providing resources and tools volunteer organizers all over the world.</p>
+
+                    <p>Since July 2014, our workshops have reached {{ attendees_sum|default_if_none:'0' }} women in {{ country_count|default_if_none:'0' }} countries and our <a href="">online tutorial</a> has been read by more than 500 000 people.</p>
+
+                    <p>We're transparent about our financial. See our <a href="{% url 'core:year_2015' %}">Annual Report for 2015</a>
```
Comment (olasitarska) [inline templates/core/donate.html]: financial -> financials?

File: templates/core/donate.html [hunk 2]
```diff
@@ -2,52 +2,139 @@
...
+            </div>
+
+            <div class="row clearfix">
+                <div class="col-md-6">
+                    <h3>Our mission:</h3>
+                    <p>Django Girls is a volunteer-run <a href="{% url 'core:foundation' %}">registered charity</a>.
+                    We use your generous donations to teach thousands of women to build websites
+                    using Python and Django by providing resources and tools volunteer organizers all over the world.</p>
+
+                    <p>Since July 2014, our workshops have reached {{ attendees_sum|default_if_none:'0' }} women in {{ country_count|default_if_none:'0' }} countries and our <a href="">online tutorial</a> has been read by more than 500 000 people.</p>
```
Comment (olasitarska) [inline templates/core/donate.html]: Missing URL in `online tutorial`

File: core/views.py [hunk 3]
```diff
@@ -144,7 +144,10 @@ def contribute(request):
 
 
 def donate(request):
-    return render(request, 'core/donate.html', {})
+    return render(request, 'core/donate.html', {
+        'country_count': Event.objects.values('country').distinct().count(),
```
Comment (olasitarska) [inline core/views.py]: I think this line is unnecessary, as the same count is already done in `context_processors` so should be available in that template as well.

**Comment by olasitarska** [comment]
This is amazing! Left just few comments :)

File: templates/core/donate.html (hunk 1 above)
Comment (patjouk) [inline templates/core/donate.html]: I've checked in a dictionary: it's finances we want to use :D

**Comment by olasitarska** [comment]
Yay!

**Pull request merged**